### ISSUE TITLE
Edge may fail to close using 0.7.5.0

### ISSUE THREAD

**Issue by etherealxx** [rcmaehl/MSEdgeRedirect#319]
### Preflight Checklist

- [X] Running `microsoft-edge:https://google.com` redirects successfully
- [X] Microsoft Edge is selected in any "How do you want to open this?" box

### Install Type

Manual Update

### Install Mode

Active Mode

### Steps to reproduce

I cannot reproduce it. And i also don't really think MSER that did this. it just so happened that i updated MSER this morning, and the only app that interact with Edge is MSER, so there's that. Just putting it right here, in case someone else got the same problem. Please close the issue if after a long time no one could replicate it.

These are logs that has more than 0kb in size
[AppGeneral.log](https://github.com/rcmaehl/MSEdgeRedirect/files/12746129/AppGeneral.log)
[URIFailures.log](https://github.com/rcmaehl/MSEdgeRedirect/files/12746128/URIFailures.log)
[AppSecurity.log](https://github.com/rcmaehl/MSEdgeRedirect/files/12746127/AppSecurity.log)

### ✔️ Expected Behavior

Just using my PC casually

### ❌ Actual Behavior

MS Edge opening and reopening over and over with no particular reason. Still doing it's thing even after MSER is killed with taskmgr. Until Edge is killed, it stops. This happen only once (so far), and it doesn't happen right when the pc is booted, instead on the middle of my using-pc session. it's so random.
https://github.com/rcmaehl/MSEdgeRedirect/assets/64251396/9e6da132-101e-4972-8c45-ccdf9e599956

### Microsoft Windows version

22H2 Build 22621.2361

### Other Software

Internet Download Manager 6.41
Free Download Manager 6.17.0.4792

**Comment by darren12345** [comment]
Affecting me too, since upgrading to 0.7.5.0.

**Comment by etherealxx** [comment]
It happens again. Replicated about 5-10 minute after booting the pc, and opening UWP facebook.

**Comment by rcmaehl** [comment]
Actually, closing as duplicate of #315 but still looking into this
